src/launch.py

```diff
@@ -21,6 +24,27 @@ def typer_interface(config_file: str):
     @staticmethod
     def launch(config):
         bpy.ops.wm.open_mainfile(filepath="src/blender_assets/CropAssets.blend")
+        wc = WeatherController("2b8fb3c4f62844189b7edec1063d92f9")
```

Comment:
Is this a secret key? If yes this should immediately be changed/revoked and GitHub secrets should be used instead.